app/Models/EntryDAO.php

```diff
@@ -960,7 +1013,7 @@ public static function sqlBooleanSearch(string $alias, FreshRSS_BooleanSearch $f
 			if ($filter->getSearch() !== null) {
```

Comment:
Looks like you're missing the `getNotInurl` case here

Reply:
Thanks! Fixed